.eslintrc.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
             rules: { '@typescript-eslint/no-var-requires': 'off' },
         },
         {
-            files: ['projects/Mallard/src/**/*'],
+            files: ['projects/?allard/src/**/*'],
```

Comment:
because on MacOS the filesystem is case insensitive and it was causing it not to show these as errors locally on my machine because it was accessing the folder as "mallard", not "Mallard". This is a bug in eslint, essentially (it should be using canonical names). This is a just a quick fix for that, `?` will match any letter so it does the trick. I can send a separate PR but I thought I could just piggyback this.

Reply:
Can you add this as a comment and maybe a link please? 👍